charts/values.yaml.tpl

```diff
@@ -104,6 +104,17 @@ infrastructure:
     guac:
       enabled: {{ $tpa.Enabled }}
       namespace: {{ $tpa.Namespace }}
+  openShiftPipelines:
+    enabled: {{ $rhdh.Enabled }}
+    patchClusterTektonConfig:
+      annotations:
+        meta.helm.sh/release-name: rhtap-backing-services
+        meta.helm.sh/release-namespace: {{ $argoCDNamespace }}
+      labels:
+        app.kubernetes.io/managed-by: Helm
+    tektonChains:
```

Comment:
Let's discuss tomorrow why that's needed.

Reply:
It's needed to replace this:

https://github.com/redhat-appstudio/rhtap-cli/blob/2cff2c50dd15a083c7f555cfd416d6f47e2d657c/charts/rhtap-backing-services/hooks/pre-deploy.sh#L6-L17

Thus we can run these type of interactions in the cluster, without the need for extra local tools to run `rhtap-cli`.